webapp/templates/home.html.tmpl

```diff
@@ -1,4 +1,17 @@
-{{ define "content" }}
-<h1>{{ .title }}</h1>
-<p>This is the home page</p>
-{{ end }}
+{{define "indexPage"}}
+<!doctype html>
+<html>
+  {{ template "header" . }}
+  <body>
+    {{ template "sidebar" . }}
+    <section class="content">
+      {{ template "submenu" . }}
+      <div class="container">
+        <h1>{{ .Title }}</h1>
+         <p>This is the home page</p>
+      </div>
+    </section>
+    {{ template "footer"}}
+  </body>
+</html>
+{{end}}
```

Comment:
This is the exact opposite of what we want: we want a generic layout where we override the "content" block, not a specific page where we duplicate all the inner block calls.

Reply:
done